src/Pandell.PackageRunner/Program.cs

```diff
@@ -245,7 +310,10 @@ private static Parameters ParseArguments(IEnumerable<string> args)
                 new { names = new [] {"username", "u"}, parse = new Action<string>(val => { parameters.RepositoryUsername = val; } )},
                 new { names = new [] {"password", "w"}, parse = new Action<string>(val => { parameters.RepositoryPassword = val; } )},
                 new { names = new [] {"config", "c"}, parse = new Action<string>(val => { parameters.Config = val; } )},
-                new { names = new [] {"disableupdates", "d"}, parse = new Action<string>(val => { parameters.DisableUpdates = true; } )}
+                new { names = new [] {"disableupdates", "d"}, parse = new Action<string>(val => { parameters.DisableUpdates = true; } )},
+                new { names = new [] {"help", "h"}, parse = new Action<string>(val => { parameters.ShowHelp = true; } )},
+                new { names = new [] {"version", "v"}, parse = new Action<string>(val => { parameters.ShowVersion = true; } )},
+                new { names = new [] {"verbose"}, parse = new Action<string>(val => { parameters.VerboseLog = true; } )}
```

Comment:
How about updating verbose flag to accept optional numeric value for level of verbosity:

```sh
PackageRunner --verbose # => TraceLevel.Verbose and below

PackageRunner --verbose:3 # => TraceLevel.Info and below

PackageRunner # => TraceLevel.Warning and below
```